draft-ietf-quic-v2.md

```diff
@@ -143,11 +145,15 @@ change to:
 
 ~~~
 secret =
-  0x3425c20cf88779df2ff71e8abfa78249891e763bbed2f13c048343d348c060e2
-key = 0xba858dc7b43de5dbf87617ff4ab253db
-nonce = 0x141b99c239b03e785d6a2e9f
+  0x66e90fdd03db030d5c80eda22a45456cf664e37f719251105953350726a609cd
+key = 0xd42ce594316b516e9db972acf7656505
+nonce = 0xac3aac3139c1f33b5c9b6091
 ~~~
 
+The secret is the sha256sum of "QUICv2 Retry Secret". The key and nonce are
```

Comment:
This is worse again than the previous ones, aside from the moved period.  I had to knock the capitals off to get the value you have.